pkg/relayer/cmd/cmd.go

```diff
@@ -0,0 +1,250 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+	"os/signal"
+
+	"cosmossdk.io/depinject"
+	cosmosclient "github.com/cosmos/cosmos-sdk/client"
+	cosmosflags "github.com/cosmos/cosmos-sdk/client/flags"
+	cosmostx "github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+
+	"github.com/pokt-network/poktroll/pkg/client/block"
+	eventsquery "github.com/pokt-network/poktroll/pkg/client/events_query"
+	"github.com/pokt-network/poktroll/pkg/client/supplier"
+	"github.com/pokt-network/poktroll/pkg/client/tx"
+	"github.com/pokt-network/poktroll/pkg/relayer"
+	"github.com/pokt-network/poktroll/pkg/relayer/miner"
+	"github.com/pokt-network/poktroll/pkg/relayer/proxy"
+)
+
+var (
+	signingKeyName string
```

Comment:
Note that lots of changes were made in #108, but I think it should be a basic `git checkout main pkg/relayer/cmd/cmd.go` after it merges in if no changes were made.

Reply:
Thanks for the heads up. It doesn't look like `pkg/relayer/cmd` was modified in #108 though so hopefully it's a non-issue. :crossed_fingers:

Reply:
Yeah, there should be no conflict

Reply:
only conflict is the root cmd